src/audio/audio_config.rs

```diff
@@ -25,13 +25,15 @@ pub struct AudioConfig {
 }
 
 impl AudioConfig {
+    /// # Safety
+    /// `handle` must be a valid handle to a live audio config.
     fn from_handle(handle: SPXAUDIOCONFIGHANDLE) -> Result<AudioConfig> {
```

Comment:
since you made remaining functions unsafe, is there any reason why this one is not unsafe as well?

Reply:
No, I just missed it.